src/modalities/nn/mlp.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+
+from torch import Tensor, nn
+
+
+class MLP(nn.Module):
+    def __init__(
+        self,
+        in_features: int,
+        hidden_features: Optional[int] = None,
+        out_features: Optional[int] = None,
+        bias: bool = True,
+        dropout: float = 0.0,
+        act_fn: nn.Module = nn.GELU,

```

Comment:
Better type hint would be Callable[[], nn.Module] to allow parameterized activation functions, e.g. "lambda: nn.LeakyReLU(negative_slope=0.02)".